tests/cypress/e2e/24_otpLogin.cy.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable global-require */
+describe('Test otp login', () => {
+  const { hguser } = require('../fixtures/user.json');
+
+  it('Test OTP login', () => {
+    cy.visit('/', { failOnStatusCode: false })
+      .get('#hgnav')
+      .find('.header__button')
+      .click();
+    cy.get('.field-username').type(hguser.email, { log: false });
+    cy.get('button[data-cy="form-submit-btn"]:visible').click();
+    cy.get('#otp').type(1234, { log: false });
```

Comment:
Not possible to capture or mock OTP in Cypress tests. OTP login can't be used here.